ethcontract/Cargo.toml

```diff
@@ -33,6 +33,7 @@ ipc-tokio = ["web3/ipc-tokio"]
 arrayvec = "0.7"
 ethcontract-common = { version = "0.15.4", path = "../ethcontract-common" }
 ethcontract-derive = { version = "0.15.4", path = "../ethcontract-derive", optional = true}
+gas-estimation = { git = "https://github.com/gnosis/gp-gas-estimation.git", tag = "v0.4.0"}
```

Comment:
Not a huge fan of pulling in a `git` dependency here (this means that we can't publish to [crates.io](https://crates.io) for example).

Maybe, instead of adding the conversion here, we can add it to `gp-gas-estimation` behind a feature (to avoid building additional dependencies that aren't needed)?

Reply:
Was hoping for this kind of a suggestion :)
So, just to confirm, put `gas_price` module in `ethcontract-rs` under feature, then import that feature in `gp-gas-estimation`?

Reply:
🙈 
Sorry I didn't see the reply. What I meant was making `ethcontract` optional in `gp-gas-estimation` so that it only pulls in the dependency if you:
```toml
gas-estimation = { version = "*", features = ["ethcontract"] }
```

With my suggestion, there were no changes required in `ethcontract.